sdk/python/requirements.in

```diff
@@ -21,3 +21,4 @@ requests_toolbelt>=0.8.0
 # CLI
 tabulate
 click
+colorama>=0.4.4
```

Comment:
What happens if colorama is not in this list?

Reply:
Based upon the [documentation](http://click.pocoo.org/5/api/#click.echo) the click.echo statement should:
```In addition to that, if colorama is installed, the echo function will also support clever handling of ANSI codes. ```

Here under [ANSI COLOR](https://click.palletsprojects.com/en/5.x/utils/) it is detailed more, seems to be implemented such that it will not break anything if not installed.

Reply:
@Ark-kun Did you get chance to look into the reference shared by @NikeNano?

Reply:
I have concern regarding this as well, mainly for the following two reasons.

1. the behavior of `colorama` is kind of tricky. We've seen a couple of weird issues/breakages due to that. For example, when using with `absl.logging`, `colorama` will cause double logging under some circumstances. This is because of the stderr redirection as stated in https://github.com/tartley/colorama/issues/209#issuecomment-465119302.

2. In general, I'm wondering if the additional maintenance efforts and possible risk of dependency conflicts outweigh the merit of a cosmetic improvement.